### DISCUSSION TITLE
救命一下，尝试下载直播流时发生错误“没有可用的直播地址”

### DISCUSSION THREAD

**Discussion by wwwu123** [HMBSbige/BilibiliLiveRecordDownLoader#315] [Q&A] [answered]
尝试下载直播流时发生错误“没有可用的直播地址”。大部分直播间可以录制，少数几个直播间会出现这个错误。

**Comment by HMBSbige** [comment]
日志提供一下呗
（猜测是部分直播间没有 flv 流，只有 HLS 流）

**Reply by wwwu123** [reply]
![螢幕擷取畫面 (1312)](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/98141895/23891dd8-c1cf-44c9-9f11-a23f0a493bf3)

**Reply by wwwu123** [reply]
flv/HLS是主播那边能改的吗？还是？

**Reply by wwwu123** [reply]
请问我要怎么整个打开这个日志？

**Reply by wwwu123** [reply]
日志点不开

**Reply by HMBSbige** [reply]
尝试设置 不同直播流格式、画质、登录账号

**Reply by HMBSbige** [reply]
我这边试了同个直播间是可以的嘛，可能是网络问题吧

你可以试试更改该直播间的高级设置针对你自己的情况调整阈值

**Reply by wwwu123** [reply]
我这边的话，同样的设置，有时候可以有时候不行。

**Reply by wwwu123** [reply]
同个直播间

**Reply by HMBSbige** [reply]
是不是录制太多了呢？
这种情况要么是B站给的限制，要么就是自身的网络问题

**Reply by wwwu123** [reply]
我一次只录一人也会出问题，网络的话看她直播也是顺的。

**Reply by HMBSbige** [reply]
试试调高 直播连接超时
或者其他设置，找到适合自己的环境吧

**Reply by wwwu123** [reply]
我调了没用，您能不能试试这个32147156，她们公会的我都录不动

**Reply by HMBSbige** [reply]
我这边都可以哦

**Reply by HMBSbige** [reply]
如果你登录+调了各种设置都没用的话，我也想不出有什么办法了

**Reply by wwwu123** [reply]
那可能是没登录的问题，我之后试试，辛苦你了作者。

**Reply by wwwu123** [reply]
确实是我没登录造成的

**Reply by wwwu123** [reply]
呃...后来还是不行，我自己慢慢调吧

**Reply by wwwu123** [reply]
日志如何完整打开？

**Reply by HMBSbige** [reply]
在程序目录里呗

**Reply by wwwu123** [reply]
人不在大陆有影响？

**Reply by HMBSbige** [reply]
可能有，具体跟B站与你自身的网络情况有关，已经不在本软件的涉及范围内了
（PS：登录后，日志记录的地址包含账号敏感信息）

**Reply by wwwu123** [reply]
好，了解

**Reply by wwwu123** [reply, marked as answer]
您好，我使用网络代理连到大陆后已经能够正常录制了，谢谢作者的开发以及解惑。
